vehicle/external_cmd_converter/src/node.cpp

```diff
@@ -113,14 +97,19 @@ void ExternalCmdConverterNode::onExternalCmd(const ExternalControlCommand::Const
   // Save received time for rate check
   latest_cmd_received_time_ = std::make_shared<rclcpp::Time>(this->now());
 
+  // take data from subscribers
+  current_velocity_ptr_ = velocity_sub_.takeData();
+  current_shift_cmd_ = shift_cmd_sub_.takeData();
+
   // Wait for input data
   if (!current_velocity_ptr_ || !acc_map_initialized_) {
```

Comment:
```suggestion
  if (!current_velocity_ptr_ || !acc_map_initialized_ || !current_shift_cmd_) {
```

Reply:
Thank you for your suggestion!

Reply:
Resolve in https://github.com/autowarefoundation/autoware.universe/pull/7263/commits/699383f576a8bea3c3f1527692fd5ffb1bf59bba